<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NicknamePage</class>
 <widget class="QWidget" name="NicknamePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="topLayout">
   <item>
    <widget class="QLabel" name="labelAlerts">
     <property name="visible">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 0, stop:0 #F0D0A0, stop:1 #F8D488); color:#000000; }</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>3</number>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>New nickname:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QValidatedLineEdit" name="NewNicknameEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Address for the nickname:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QValidatedLineEdit" name="NickAddressEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="RegisterNicknameBtn">
         <property name="text">
          <string>Register new nickname for your address</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QValidatedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qt/qvalidatedlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
